.github/test-categories/MISC

```diff
@@ -0,0 +1,6 @@
+com.ibm.ws.cloudant_fat
```

Comment:
We no longer support building with github actions ... so this file should be deleted